lib/segment/src/types.rs

```diff
@@ -253,6 +253,27 @@ pub struct SegmentInfo {
     pub index_schema: HashMap<PayloadKeyType, PayloadIndexInfo>,
 }
 
+/// Additional parameters of the search
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone, Copy, PartialEq, Eq, Default)]
+#[serde(rename_all = "snake_case")]
+pub struct QuantizationSearchParams {
+    /// If true, quantized vectors are ignored. Default is false.
+    #[serde(default = "default_quantization_ignore_value")]
+    pub ignore: bool,
+
+    /// If true, use original vectors to re-score top-k results. Default is true.
```

Comment:
`default_quantization_rescore_value` is false. even if it's todo task, I propose to change the comment to `Default is false`